app/models/user.rb

```diff
@@ -1,10 +1,15 @@
 class User < ApplicationRecord
-  has_many :posts, foreign_key: :author_id
-  has_many :comments
-  has_many :likes
+  # Include default devise modules. Others available are:
+  # :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] According to the project requirements, the application should ask for confirmation of email after the sign-up request.
- Great job enabling the `:confirmable` in the user model (`user.rb` file). 👏🏻  However, this prevents the user to be created in the db. to fix this, you need to setup correctly the  confirmation email user email in your app. here is the step you need to follow: 
1. Uncomment  `:confirmable` in the user model.
2. Update your AddDeviseToUsers by uncommenting these line:

```ruby 
      ## Confirmable
      t.string   :confirmation_token
      t.datetime :confirmed_at
      t.datetime :confirmation_sent_at
      t.string   :unconfirmed_email # Only if using reconfirmable

     add_index :users, :confirmation_token,   unique: true

```
3. Then run a new migration `rails db:drop db:create db:migrate`
After that try to sign_up then check your console.log Devise will send you a email confirmable to confirm your account.

- Kindly follow this step 💯 👍🏻  If you need help feel free to reach out to me on Slack or tag me here 👍🏻